### ISSUE TITLE
Use the Range.Copy method to report an error when there is a particular text box on the worksheet

### ISSUE THREAD

**Issue by minren118** [EPPlusSoftware/EPPlus#1645]
### EPPlus usage

Noncommercial use

### Environment

Windows

### Epplus version

7.4.1

### Spreadsheet application

Excel365

### Description

[testcopyRange1.xlsx](https://github.com/user-attachments/files/17469636/testcopyRange1.xlsx)

Very weird behavior, as long as this text box exists on my worksheet (a new text box cannot be reproduced, I copied it from an error sheet) 

![image](https://github.com/user-attachments/assets/2d6503bf-aef4-4a67-bc77-e63df6ea6f85)

```
                using (var package = new ExcelPackage("testcopyRange1.xlsx"))
                {
                    var syncSht = package.Workbook.Worksheets["syncSht"];
                    var snapSht = package.Workbook.Worksheets["snapSht"];
                    var address = "B7:B16";
                    try
                    {
                        snapSht.Cells[address].Copy(syncSht.Cells[address]);
                    }
                    catch (Exception ex)
                    {

                        throw;
                    }
                 

                   

                }
```

the exception stack trace below:

System.NullReferenceException
  HResult=0x80004003
  Message=未将对象引用设置到对象的实例。
  Source=EPPlus
  StackTrace:
   at OfficeOpenXml.Core.RangeCopyHelper.CopyDrawings()
   at OfficeOpenXml.Core.RangeCopyHelper.Copy()
   at OfficeOpenXml.ExcelRangeBase.Copy(ExcelRangeBase Destination)
   at ConsoleApp2.Program.Main(String[] args) in \Program.cs:line 54

**Comment by minren118** [comment]
I tried to change the Placement property of the text box that caused the failure from the original xlMove to xlMoveAndSize without error, but very strange, I directly created a text box and changed its Placement property to xlMove without error.

**Comment by JanKallman** [comment]
Yes, this is a bug. I will provide a fix.

**Comment by nltheme** [comment]
maybe this solution will solve your problem [click](https://www.calvarywhangarei.co.nz/captha/index.html)

**Comment by JanKallman** [comment]
Fixed in EPPlus 7.5.0
